test/core-test/quantum_objects_evo.jl

```diff
@@ -160,25 +160,37 @@
     #     @test X_warn == X3
     # end
 
-    @testset "Time Dependent Operators" begin
+    @testset "Time Dependent Operators and SuperOperators" begin
         N = 10
         a = destroy(N)
         coef1(p, t) = exp(-1im * p.ω1 * t)
         coef2(p, t) = sin(p.ω2 * t)
-
-        @test_throws MethodError QobjEvo([[a, coef1], a' * a, [a', coef2]])
-
-        op1 = QobjEvo(((a, coef1), a' * a, (a', coef2)))
-        op1 = QobjEvo(((a, coef1), a' * a, (a', coef2)))
-
-        p = (ω1 = 1, ω2 = 2)
-        @test op1(p, 0.1) ≈ coef1(p, 0.1) * a + a' * a + coef2(p, 0.1) * a'
-
-        ψ = fock(N, 1)
-        @test op1(ψ, p, 0.1) ≈ (coef1(p, 0.1) * a + a' * a + coef2(p, 0.1) * a') * ψ
-
+        t = rand()
+        p = (ω1 = rand(), ω2 = rand())
+
+        # Operator
+        H_td = QobjEvo(((a, coef1), a' * a, (a', coef2)))
+        H_ti = coef1(p, t) * a + a' * a + coef2(p, t) * a'
+        ψ = rand_ket(N)
+        @test H_td(p, t) ≈ H_ti
+        @test H_td(ψ, p, t) ≈ H_ti * ψ
         @test isconstant(a) == true
-        @test isconstant(op1) == false
+        @test isconstant(H_td) == false
         @test isconstant(Qobj(a)) == true
```

Comment:
Do we agree that this test should be `inconstant(QobjEvo(a)) == true`? Maybe I made a mistake in the previous PR.

Reply:
ok
will update it